<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDlg</class>
 <widget class="QDialog" name="SettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>119</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>236</horstretch>
    <verstretch>119</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edit_address">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>9</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_port">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>35</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_dbname">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>61</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="address">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес сервера:</string>
   </property>
  </widget>
  <widget class="QLabel" name="port">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dbname">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>61</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя БД:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
